nix/devshell.nix

```diff
@@ -19,6 +20,7 @@
         pkg-config
         python3
         python3Packages.pip
```

Comment:
Could you please remove these two lines? python3 and python3Packages.pip

Reply:
done